### DISCUSSION TITLE
The difference bwteen this project and nexus-plugin-prisma

### DISCUSSION THREAD

**Discussion by hantsy** [prisma/nexus-prisma#65] [General]
I found there is another project under nexus: https://github.com/graphql-nexus/nexus-plugin-prisma

What is the difference?

**Comment by dominichadfield-jelly** [comment]
That is a repo for the old plugin, support is being dropped in favour of the new plugin and, as a result, support for newer versions of prisma are not supported.

This repo is a complete rewrite but as a result does not have all the functionality of the old one and the api is drastically different.

I recommend waiting for this version to be ready or if you cannot wait, starting without the plugin and incrementally adopting it as the features are built out. I would not recommend using the old plugin as it will require a large rewrite anyway.

**Reply by janpio** [reply]
The whole thing is explained in https://github.com/graphql-nexus/nexus-plugin-prisma/issues/1039
